homeassistant/components/vallox/services.yaml

```diff
@@ -27,3 +27,23 @@ set_profile_fan_speed_boost:
           min: 0
           max: 100
           unit_of_measurement: "%"
+
+set_profile:
+  fields:
+    profile:
+      required: true
+      selector:
+        select:
+          options:
+            - "Home"
+            - "Away"
+            - "Boost"
+            - "Fireplace"
+            - "Extra"
```

Comment:
Can we use lower case here and use translations?

Reply:
Quite a big change in this context I think, could do separately if there's some guidance for this

Reply:
It's not that big of a change, but I would like to have it in this PR to avoid a migration path in the future.

I think a global search for "selectors" in a "strings.json" file would probably yield some results. I'm currently on mobile so can't check

Reply:
Looked a bit at this, and everything currently in the module code seems to deal with values like 'Home'

just changing this selector to use translations, would mean an extra mapping `home` -> `Home` -> `vallox_websocket_api.Profile.HOME` for this

and i guess all previous state data is stored with 'Home' etc, so not sure what this change would accomplish

can do what's needed but need some guidance

Reply:
This change will add the ability to be translated. This increases usability for non English speakers. You would have to map the values between lower case and what the library gives and returns